src/js/utils/styles.js

```diff
@@ -25,6 +25,14 @@ export const baseStyle = css`
   -webkit-font-smoothing: antialiased;
 `;
 
+export const borderStyle = css`
```

Comment:
Can we call this `controlBorderStyle` to be more explicit?

Reply:
sure :)